spec/controllers/projects_controller_spec.rb

```diff
@@ -301,20 +301,21 @@
     end
   end
 
-  describe 'GET /:episode/projects.rss' do
+  describe 'GET /:episode/projects.rss' do    
     context '.rss' do
       # We want to parse the outputted XML, so let's turn off rendering stubbing
       render_views
-
       # We are creating our helpers eagerly, so they are in the DB at the request time
       let!(:episode) { create :episode }
-      let!(:old_projects) do
-        Timecop.freeze(1.year.ago) do
-          (1..12).map { create :project, episodes: [episode] }
-        end
-      end
+
       let!(:new_projects) { (1..10).map { create :project, episodes: [episode] } }
 
+      # let!(:old_projects) do
```

Comment:
Did you forget to remove this comments?

Reply:
actually, I didn't found any use of that code block, so commented it to resolve an error in line 329(since projects aren't ordered in desc created_at) :sweat_smile: